src/flb_storage.c

```diff
@@ -532,6 +532,18 @@ int flb_storage_input_create(struct cio_ctx *cio,
             return -1;
         }
     }
+    else if (stream->type != cio_storage_type) {
+        flb_info("[storage] storage type mismatch. input type=%s",
```

Comment:
I will put this as a debug instead of info

Reply:
In this case user environment is need to be fixed.
- Remove storage
- Modify storage property

I think it is better the level is info or warn to notice it.
What do you think ?

Reply:
hmm, I would not remove the storage content for the user unless the directory is empty (no chunks), but I agree with better messaging.

Reply:
I modified log message. Could you check it ?
https://github.com/fluent/fluent-bit/pull/8290/commits/34c00a1f3bbcea33317e5a8f9d5e37ed087cb17d